lava-v2-jobs-from-api.py

```diff
@@ -156,7 +156,7 @@ def get_job_params(config, template, opts, device, build, defconfig, plan):
 
     if 'kselftest' in plan:
         initrd_url = KSELFTEST_INITRD_URL.format(initrd_arch)
-    elif 'v4l2' in plan:
+    elif 'v4l2' or 'igt' in plan:
```

Comment:
It's OK for now but I think we really need to revisit all this logic and remove any reference to test plan names in this script before adding even more test plans.